x-pack/legacy/plugins/siem/public/containers/case/use_get_case_user_actions.test.tsx

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { renderHook, act } from '@testing-library/react-hooks';
+import {
+  initialData,
+  useGetCaseUserActions,
+  UseGetCaseUserActions,
+} from './use_get_case_user_actions';
+import { basicCaseId, caseUserActions, elasticUser } from './mock';
+import * as api from './api';
+
+jest.mock('./api');
+
+describe('useGetCaseUserActions', () => {
+  const abortCtrl = new AbortController();
+  beforeEach(() => {
+    jest.clearAllMocks();
+    jest.restoreAllMocks();
+  });
+
+  it('init', async () => {
+    await act(async () => {
+      const { result, waitForNextUpdate } = renderHook<string, UseGetCaseUserActions>(() =>
+        useGetCaseUserActions(basicCaseId)
+      );
+      await waitForNextUpdate();
+      expect(result.current).toEqual({
+        ...initialData,
+        fetchCaseUserActions: result.current.fetchCaseUserActions,
+      });
+    });
+  });
+
+  it('calls getCaseUserActions with correct arguments', async () => {
+    const spyOnPostCase = jest.spyOn(api, 'getCaseUserActions');
+
+    await act(async () => {
+      const { result, waitForNextUpdate } = renderHook<string, UseGetCaseUserActions>(() =>
+        useGetCaseUserActions(basicCaseId)
+      );
+      await waitForNextUpdate();
+
+      result.current.fetchCaseUserActions(basicCaseId);
+      await waitForNextUpdate();
+      expect(spyOnPostCase).toBeCalledWith(basicCaseId, abortCtrl.signal);
+    });
+  });
+
+  it('retuns proper state on getCaseUserActions', async () => {
+    await act(async () => {
+      const { result, waitForNextUpdate } = renderHook<string, UseGetCaseUserActions>(() =>
+        useGetCaseUserActions(basicCaseId)
+      );
+      await waitForNextUpdate();
+      result.current.fetchCaseUserActions(basicCaseId);
+      await waitForNextUpdate();
+      expect(result.current).toEqual({
+        ...initialData,
+        caseUserActions: caseUserActions.slice(1),
+        fetchCaseUserActions: result.current.fetchCaseUserActions,
+        hasDataToPush: true,
+        isError: false,
+        isLoading: false,
+        participants: [elasticUser],
+      });
+    });
+  });
+
```

Comment:
Maybe we can add a test that validates the correctness of `getPushedInfo` function. What do you think?

Reply:
this function is actually covered minus one case. im going to let it go in favor of feature work:
<img width="813" alt="Screen Shot 2020-04-14 at 10 23 18 AM" src="https://user-images.githubusercontent.com/6935300/79249101-feb91700-7e39-11ea-993a-197e22a3e506.png">